code/clue.js

```diff
@@ -7,53 +7,281 @@ const mrGreen = {
+  image: 'assets/knife.png',
+}
+
+const candlestick = {
+  name: 'Candlestick',
+  weight: 6,
+  image: 'assets/candlestick.png',
+}
+
+const dumbbell = {
+  name: 'Dumbbell',
+  weight: 20,
+  image: 'assets/dumbbell.png',
+}
+
+const poison = {
+  name: 'Poison',
+  weight: 1,
+  image: 'assets/poison.png',
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const axe = {
+  name: 'Axe',
+  weight: 30,
+  image: 'assets/axe.png',
+}
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const bat = {
+  name: 'bat',
+  weight: 25,
+  image: 'assets/bat.png',
+}
+
+const trophy = {
+  name: 'Trophy',
+  weight: 12,
+  image: 'assets/trophy.png',
+}
+
+const pistol = {
+  name: 'Pistol',
+  weight: 7,
+  image: 'assets/pistol.jpg',
+}
+
+// THE ROOMS 
+
+const diningRoom = {
+  name: 'Dining room',
+  image: 'assets/diningroom.jpg',
+}
+
+const conservatory = {
+  name: 'Conservatory',
+  image: 'assets/conservatory.png',
+}
+
+const kitchen = {
+  name: 'Kitchen',
+  image: 'assets/kitchen.png',
+}
+
+const study = {
+  name: 'study',
+  image: 'assets/study.png',
+}
+
+const library = {
+  name: 'Library',
+  image: 'assets/library.png',
+}
+
+const billiardRoom = {
+  name: 'Billiard room',
+  image: 'assets/billiardroom.png',
+}
+
+const lounge = {
+  name: 'Lounge',
+  image: 'assets/lounge.jpg',
+}
+
+const ballroom = {
+  name: 'Ballroom',
+  image: 'assets/ballroom.png',
+}
+
+const hall = {
+  name: 'Hall',
+  image: 'assets/hall.jpg',
+}
+
+const spa = {
+  name: 'Spa',
+  image: 'assets/spa.jpg',
+}
+
+const livingRoom = {
+  name: 'Living room',
+  image: 'assets/livingroom.jpg',
+}
+
+const observatory = {
+  name: 'Observatory',
+  image: 'assets/observatory.png',
+}
+
+const theater = {
+  name: 'Theater',
+  image: 'assets/theater.jpg',
+}
+
+const guestHouse = {
+  name: 'Guest house',
+  image: 'assets/guesthouse.png',
+}
+
+const patio = {
+  name: 'Patio',
+  image: 'assets/patio.jpg',
+}
+
+// ARRAYS
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
+  prPlum,
+  mrsScarlet,
+  colMustard,
+  mrsPeacock,
+  mrsWhite,
 ]
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol,
+]
 
-const rooms = []
+const rooms = [
+  diningRoom,
+  conservatory,
+  kitchen,
+  study,
+  library,
+  billiardRoom,
+  lounge,
+  ballroom,
+  hall,
+  spa,
+  livingRoom,
+  observatory,
+  theater,
+  guestHouse,
+  patio,
+]
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+// OBJECT THAT KEEPS THE MYSTERY.
+const mystery = {
+  killer: '',
+  weapon: '',
+  room: '',
```

Comment:
Super small thing, but I would suggest you to initialise your `mystery` object with `null` values rather than empty strings. It's about sending proper information for other developers : `null` value indicates that probably we will replace these values with something else, but without saying what type will that new value have for now. Initialising your object with empty string values `' '` could be misunderstood by some one in a way that your properties will later on by replaced by new strings only. As a sum up, this would be better :
```
const mystery = {
  killer: null,
  weapon: null,
  room: null,
}
```